src/main/java/com/easyrest/dao/OrderDao.java

```diff
@@ -0,0 +1,94 @@
+package com.easyrest.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class OrderDao {
+    public static final String USER_ID_BY_EMAIL_QUERY = "SELECT id " +
+            "FROM users " +
+            "WHERE email=?";
+    public static final String RESTAURANT_ID_BY_ADMINISTRATOR_ID_QUERY = "SELECT id " +
+            "FROM restaurants " +
+            "WHERE administrator_id=?";
+    public static final String INSERT_WAITING_FOR_CONFIRM_ORDER_QUERY = "INSERT " +
+            "INTO orders(status,user_id,rest_id) " +
+            "VALUES ('Waiting for confirm','27',?) " +
+            "RETURNING id";
+    public static final String INSERT_ACCEPTED_ORDER_QUERY = "INSERT " +
+            "INTO orders(status,user_id,rest_id) " +
+            "VALUES ('Accepted','27',?) " +
+            "RETURNING id";
+    public static final String DELETE_ORDER_QUERY = "DELETE " +
+            "FROM orders " +
+            "WHERE id=?";
+
+    public int saveWaitingForConfirmOrder(String email) {
+        int insertedOrderId = 0;
+        try (Connection connection = DBManager.openConnection()) {
+            int restaurantId = getRestaurantId(connection, email);
+            try (PreparedStatement addOrderStatement = connection.prepareStatement(INSERT_WAITING_FOR_CONFIRM_ORDER_QUERY)) {
+                addOrderStatement.setInt(1, restaurantId);
+                ResultSet addOrderResultSet = addOrderStatement.executeQuery();
+                while (addOrderResultSet.next()) {
+                    insertedOrderId = addOrderResultSet.getInt(1);
+                }
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
should you stop test execution at this point?

Reply:
Fixed.